price-feeder/go.mod

```diff
@@ -4,17 +4,20 @@ go 1.17
 
 require (
 	github.com/BurntSushi/toml v0.4.1
-	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/armon/go-metrics v0.3.10
```

Comment:
Pinned to the most recent version to include the `DenomList.Contains` method introduced in #418